src/opentimelineio/composition.h

```diff
@@ -23,7 +23,7 @@ class Composition : public Item {
 
     virtual std::string const& composition_kind() const;
```

Comment:
```suggestion
    virtual std::string composition_kind() const;
```

Reply:
I was planning on submitting a separate PR for const cleanup changes, though I guess it could be part of this one? There's a number of these that could be changed.

Reply:
I'm inclined to rip off the bandaid and get it done as part of constexpr just because we have to address general constness with this PR. a change to C++ version is a separate concern, so it made sense to me to split it. I don't see constness as meaningfully separable though.